api/HTMLElement.json

```
@@ -570,14 +570,62 @@
               "version_added": null
             },
             "webview_android": {
-              "version_added": null
+              "version_added": "45"
             }
           },
           "status": {
             "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "options": {
+          "__compat": {
+            "description": "Supports the <code>preventScroll</code> boolean option.",
```

No full stop at the end here, please

```suggestion
            "description": "Supports the <code>preventScroll</code> boolean option",
```